.gitignore

```
@@ -1,67 +1,55 @@
-# git ignore patterns
 
-libltdl/
-
-pynest/pynestkernel.cpp
+__pycache__/
+build/
+install/
+reports/
 
-*.o
-*.lo
+# compilation artefacts
+*.cxx
```

I think excluding *.cxx files in a C++ project is dangerous, so I suggest to remove this. What did it apply to?

---

This is meant to exclude the C++ file generated by Cython and I agree that the pattern is way too unspecific. What about the following change?
```suggestion
pynest/pynestkernel.cxx
```

---

@jougs I like you suggestion to give the full path!